EpiAware/src/observation-processes.jl

```diff
@@ -6,14 +6,13 @@ end
         y_t,
         I_t,
         epimodel::AbstractEpiModel;
-        observation_process_priors = default_delay_obs_priors(),
-        pos_shift = 1e-6
+        kwargs...
```

Comment:
hm this isn't quite what I had in mind. Can you not specify the precise arguments here (i.e neg_bin_cluster_factor_prior) and then use splatting when calling the function to pattern match to these arguments? This doesn't really help docwise etc?

Reply:
Yes you can!

This was a mini brain fade.